frontend/apps/filemanager/filemanager.lua

```diff
@@ -1279,6 +1285,36 @@ function FileManager:showFiles(path, focused_file)
     UIManager:show(file_manager)
 end
 
+function FileManager:openTextViewer(file_path)
+    local function _openTextViewer(filepath)
+        local file = io.open(filepath, "rb")
+        if not file then return end
+        local file_content = file:read("*all")
+        file:close()
+        UIManager:show(require("ui/widget/textviewer"):new{
+            title = filepath,
+            title_multilines = true,
+            justified = false,
+            text = file_content,
+        })
+    end
+    local attr = lfs.attributes(file_path)
+    if attr then
+        if attr.size > 400000 then
+            UIManager:show(ConfirmBox:new{
+                text = T(_("This file is %2:\n\n%1\n\nAre you sure you want to open it?\n\nOpening big files may take some time."),
```

Comment:
Just double checking, this is the exact same string as in text editor? (So it's already translated.)

Reply:
Copy-paste from Text editor.